desktop/api/src/main/java/org/datacleaner/util/UserWindowSizePreferencesUtils.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.util;
+
+import java.awt.Dimension;
+import java.util.Map;
+
+import org.datacleaner.user.UserPreferences;
+
+public class UserWindowSizePreferencesUtils {
```

Comment:
Sorry to keep bothering about this point, but I still feel the name of the class is kinda weird. It is somehow referring to the UserPreferences object but I think that is a technical detail (which is revealed by the constructor argument already). We have a similar class in JobGraphPreferencesPanel. So maybe we could name this WindowSizePreferences? Then the "User" and "Utils" part is stripped, but those are also for me the least descriptive parts of the name.